packages/ses/src/module-instance.js

```diff
@@ -40,8 +51,9 @@ export const makeModuleInstance = (
   // object (eventually proxied).
   const exportsProps = create(null);
 
-  // {_localName_: accessor} added to endowments for proxy traps
-  const trappers = create(null);
+  // {_localName_: accessor} proxy traps for globalThis, globalLexicals, and
+  // live bindings.
+  const localObject = create(null, getOwnPropertyDescriptors(globalLexicals));
```

Comment:
I don't understand. In compartment-shim line 173 you copy the `globalLexicals` with `...` which leaves accessor behind. Why would the `globalLexicals` here contain accessors?

Which brings up a more basic question. Once `evaluator` has no `endowments` option, how does the client code pass in per-module (per evaluation?) accessors?

Reply:
Todo: `{ __proto__: null, ...globalLexicals }`

Reply:
As discussed, `globalLexicals` should be copied to initialize the `localObject` using `...`